2025/linux/solutions.md/task6.md

```
@@ -0,0 +1,80 @@
+# Print end status message.
+echo
+echo "Backup finished"
+date
+    
+ # Long listing of files in $dest to check file sizes.
+ls -lh $dest
+
+whenever we start to write the shell scripts we should start it with #!/bash/bin. we can use simple text editor or command line editor in order to create or edit the bash file . we use vim editor to create or edit the bash file.
+the name of our backup file will be backup_scripts.sh . the .sh extension should be present at the end of the bash file.
+
+#! /bin/bash # First Line of bash script
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)